resources/views/cart/cart.blade.php

```diff
@@ -1,2 +1,11 @@
-@include('rapidez-ct::cart.partials.products')
-@include('rapidez-ct::cart.partials.bottom-bar')
+<x-rapidez-ct::sections class="!mt-0">
+    @include('rapidez-ct::cart.partials.products')
+</x-rapidez-ct::sections>
```

Comment:
Could you elaborate what the purpose of putting this in a `sections` component does here? Given that the actual component you include here only has one div this seems unnecessary?

I can see it's the same as in the checkout-theme, maybe it's more useful to ask why it was changed to begin with?

Reply:
It's because of the margin top and bottom that being used inside of the section component. Just like you mentioned in my [other PR](https://github.com/rapidez/checkout-theme/pull/67) the margin shouldn't be set on that element but it should be set by the section component. This divides the title, sections and bottom bar visually on all the steps.